### ISSUE TITLE
Need Help with Splitting the Selection into Lines

### ISSUE THREAD

**Issue by cvladan** [dail8859/LuaScript#99]
Can someone please help me?
I need assistance with a script that would provide the same functionality as in Sublime Text or VSCode. 

Specifically, the VSCode command "insertCursorAtEndOfEachLineSelected"
originally from Sublime Text: "Splitting the Selection into Lines", see here: [Sublime Text Documentation](https://www.sublimetext.com/docs/multiple_selection_with_the_keyboard.html#splitting-the-selection-into-lines)

Simply put, I don't understand how to iterate through the lines of the current selection, regardless of whether it's a multi-edit or a classic single edit.

Thanks!

P.S. And let me not forget, thanks to the author for the useful plugin!

**Comment by dail8859** [comment]
You might be interested in my other plugin, [BetterMultiSelection](https://github.com/dail8859/BetterMultiSelection)

If you have a rectangular selection (a normal selection won't work) you can use the keyboard navigation as expected, such as `End`, `Home`, `Left/Right`, `Shift+...` and it will create multiple cursors.

![](https://raw.githubusercontent.com/dail8859/BetterMultiSelection/master/img/Demo.gif)

Just figured I'd point this out real quick...don't have access to try out any Lua code right now.
